src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -998,8 +974,8 @@ private static KustoConnectionStringBuilder CreateConnectionStringBuilder(Connec
                         }
 
                         info.ConnectionDetails.ApplicationName = "sqlops-connection-string";
```

Comment:
The name here doesn't really seem useful - and it definitely shouldn't be using sqlops anymore. Something to consider updating while you're modifying this code.